contracts/contracts/interfaces/IRewardManager.sol

```diff
@@ -3,6 +3,15 @@ pragma solidity ^0.8.0;
 import "./IAllowList.sol";
 
 interface IRewardManager is IAllowList {
+  // RewardAddressChanged is the event logged whenever reward address is modified
+  event RewardAddressChanged(address indexed sender, address indexed rewardAddress);
+
+  // FeeRecipientsAllowed is the event logged whenever fee recipient is modified
+  event FeeRecipientsAllowed(address indexed sender);
+
+  // RewardsDisabled is the event logged whenever rewards are disabled
+  event RewardsDisabled(address indexed sender);
```

Comment:
I'm not sure if `sender` addresses would be very helpful to be an indexed topic in logs. Maybe we should omit emitting those. cc @ava-labs/platform-evm @abi87

Reply:
We decided to include them